src/cpdb/helpers.py

```diff
@@ -1,10 +1,17 @@
+from distutils.command.build import build
 import pandas as pd
 from dotenv import load_dotenv
 import geopandas as gpd
 from src.digital_ocean_client import DigitalOceanClient
 
 load_dotenv()
 
+cpdb_published_version = [
```

Comment:
Better variable name is `cpdb_published_versions`

Reply:
This was changed in the helper file but not the cpdb file. Better to use VSCode's rename feature which should rename references too